ui/components/multichain/product-tour-popover/product-tour-popover.js

```diff
@@ -111,7 +111,9 @@ export const ProductTour = ({
             color={TextColor.infoInverse}
             variant={TextVariant.bodyMd}
           >
-            {currentStep}/{totalSteps}
+            {currentStep && totalSteps
+              ? { currentStep } / { totalSteps }
```

Comment:
I'm seeing `NaN`. I think this should have been `${currentStep}/${totalSteps}`.

Reply:
resolved in #23100